rb/Gemfile.lock

```diff
@@ -51,11 +51,9 @@ GEM
     i18n (1.14.1)
       concurrent-ruby (~> 1.0)
     io-console (0.7.2)
-    io-console (0.7.2-java)
```

Comment:
This would need to be reverted to make sure the CI passes on JRuby.

Reply:
Reverted